docs/Reference/Using-JSON-RPC-API.md

```diff
@@ -27,19 +27,28 @@ Use the button to import our collection of examples to [Postman](https://www.get
 
 [![Run in Postman](https://run.pstmn.io/button.svg)](https://app.getpostman.com/run-collection/cffe1bc034b3ab139fa7)
 
-### Endpoint Address and Port
+### Endpoint Host and Port
 
-In this reference, the placeholder
+The placeholder
 `<JSON-RPC-http-endpoint:port>` and `<JSON-RPC-ws-endpoint:port>` means an endpoint (IP address and port) 
-of the JSON-RPC service of a Pantheon node respectively for http endpoint and for WebSocket endpoint.
+of the JSON-RPC service of a Pantheon node for HTTP and WebSocket requests.
 
-To activate JSON-RPC using http or WebSocket, see [`--rpc-http-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-enabled) 
-and [`--rpc-ws-listen`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-listen) options in the CLI documentation.
+To enable JSON-RPC over HTTP or WebSockets, use the [`--rpc-http-enabled`](../Reference/Pantheon-CLI-Syntax.md#rpc-http-enabled) 
+and [`--rpc-ws-listen`](../Reference/Pantheon-CLI-Syntax.md#rpc-ws-listen) options.
```

Comment:
Listen for Rpc http, Rpc ws and P2P option was split in 2: host and port
And here you probably meant rpc-ws-enabled

Reply:
Updated